source/faq.txt

```diff
@@ -25,6 +25,23 @@ If you have trouble connecting to a MongoDB deployment, see
 the :ref:`Connection Troubleshooting Guide <node-connection-troubleshooting>`
 for possible solutions.
 
+What is the difference between the Node.js Driver and Mongoose?
+---------------------------------------------------------------
+
+The MongoDB Node.js driver is the officially developed, supported, and maintained
+MongoDB driver for Node.js. The Node.js driver is a library of functions
+that you can use to connect to and communicate with MongoDB.
+
+Mongoose is a popular Node.js-based third-party Object Data Modeling
+(ODM) library for MongoDB. The problem that Mongoose aims to solve is
+allowing developers to enforce a specific schema at the application
+layer. In addition to enforcing a schema, Mongoose also offers a variety
+of hooks, model validation, and other features aimed at making it easier
+to work with MongoDB. 
```

Comment:
Delete from "The problem..." to the end of the paragraph. There should be one comparison sentence here about the key difference.